client/iOS/FreeRDP/ios_freerdp.h

```diff
@@ -69,3 +81,4 @@ void ios_uninit_freerdp(void);
 freerdp *ios_freerdp_new(void);
 int ios_run_freerdp(freerdp *instance);
 void ios_freerdp_free(freerdp *instance);
+void ios_send_clipboard_data(void *context, const void* data, UINT32 size);
```

Comment:
Please add the missing new line after that added line.

Reply:
Will do!